feed.go

```diff
@@ -152,21 +151,16 @@ func (feed *ArtifactsFeed) GetLatestArchive(series string, options BuildOptions)
 		return "", errors.Wrapf(err, "problem fetching download information for series '%s'", series)
 	}
 
-	// if it's a dev version: then the branch name is in the file
-	// name, and we just take the latest from master
-	seriesNum, err := strconv.Atoi(string(series[2]))
+	isDev, err := version.isNightlyOrContinuous()
 	if err != nil {
-		// this should be unreachable, because invalid
-		// versions won't have yielded results from the feed
-		// op
-		return "", errors.Wrapf(err, "version specification is invalid")
+		return "", errors.Wrap(err, "problem determining version type")
 	}
 
-	if seriesNum%2 == 1 {
+	// If it's nightly or continuous, we just replace the version with the word latest.
```

Comment:
This method is kind of odd and I don't know what to do with it.

I don't actually think (but I'm not really certain) if continuous is supposed to be converted to the latest version of master - it seems to me like a continuous release should not be treated the same as a legacy development series release. For example, if `releases = ["4.0-latest", "4.3-latest", "5.0-latest", "5.1-latest"]`, I would have expected something like this:

`https://downloads.mongodb.com/.../...v4.0-latest.tgz` (because it's a production series)
`https://downloads.mongodb.com/.../...latest.tgz` (i.e. the latest on master, because we would like the latest for the development series but because we don't branch development series, only the latest release on the mongodb-mongo-master is available)
`https://downloads.mongodb.com/.../...v5.1-latest.tgz` (because it's an LTS series)
`https://downloads.mongodb.com/.../...v5.1-latest.tgz` (because it's continuous?)

Reply:
I guess I'm not really sure what any of his branching means. I assumed that since continuous releases were going in the same bucket as development that they should be treated like that. I see what you're saying though. This implies that any series from the new version will use the `v<major.minor>-latest` format, unless it's possible that releases could have something like ["5.6-latest"] in which case the bond IsContinuous() should probably handle for minor < 3, but I'm assuming we can trust the input.

Reply:
The branches refer to the project dedicated solely to each production series (e.g. "branching" master when doing this year's first release meant creating the mongodb-mongo-v4.4 branch). So when it's referring to something like 4.4-latest, it means the latest on mongodb-mongo-v4.4.

Kelsey says they plan to continue branching for every new LTS/continuous release, so there should be a mongodb-mongov5.0, mongodb-mongo-v5.1, etc. (which implies that "-latest" should work for all new release series unless the release team changes their mind).